app/src/main/res/layout/activity_multimeter.xml

```diff
@@ -545,83 +157,352 @@
-        app:layout_constraintTop_toTopOf="@+id/knob_line9" />
+        app:layout_constraintTop_toTopOf="@+id/knob_line_center2" />
 
     <TextView
-        android:id="@+id/knob_center"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_centerInParent="true"
-        android:layout_marginBottom="@dimen/multimeter_knob_text_margin_bottom"
-        android:layout_marginEnd="@dimen/multimeter_knob_text_margin_right"
-        android:layout_marginLeft="@dimen/multimeter_knob_text_margin_left"
-        android:layout_marginRight="@dimen/multimeter_knob_text_margin_right"
-        android:layout_marginStart="@dimen/multimeter_knob_text_margin_left"
-        android:layout_marginTop="@dimen/multimeter_knob_text_margin_top"
-        android:layout_weight="1"
-        android:background="@color/colorPrimaryDark"
-        android:text="@string/id1"
-        android:textColor="@color/white"
-        android:textSize="@dimen/multimeter_textsize"
-        app:layout_constraintBottom_toBottomOf="@+id/knob_line9"
-        app:layout_constraintEnd_toEndOf="@+id/view2"
-        app:layout_constraintStart_toStartOf="@+id/knob_line"
-        app:layout_constraintTop_toTopOf="@+id/knob_line9"
-        app:layout_constraintVertical_bias="0.489" />
+        android:layout_gravity="center"
+        android:gravity="center"
+        android:text="@string/voltage_elements_abbreviation"
+        android:textColor="@color/black"
+        android:textSize="@dimen/control_textsize_tiny"
+        app:layout_constraintCircle="@id/knobs"
+        app:layout_constraintCircleAngle="336"
+        app:layout_constraintCircleRadius="@dimen/multimeter_knobcircle_radius_2" />
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:gravity="center"
+        android:text="@string/frequency_abbreviation"
+        android:textColor="@color/black"
+        android:textSize="@dimen/control_textsize_tiny"
+        app:layout_constraintCircle="@id/knobs"
+        app:layout_constraintCircleAngle="24"
+        app:layout_constraintCircleRadius="@dimen/multimeter_knobcircle_radius_2" />
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/voltage_channels_abbreviation"
+        android:textColor="@color/black"
+        app:layout_constraintCircle="@id/knobs"
+        app:layout_constraintCircleAngle="288"
+        app:layout_constraintCircleRadius="@dimen/multimeter_knobcircle_radius_1" />
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="center"
+        android:gravity="center"
+        android:text="@string/resistance_abbreviation"
+        android:textColor="@color/black"
+        android:textSize="18sp"
```

Comment:
This is a large file so you have missed few places where you can use resource files.